sencha-workspace/SlateAdmin/app/view/people/details/Courses.js

```diff
@@ -38,23 +38,41 @@ Ext.define('SlateAdmin.view.people.details.Courses', {
         xtype: 'grid',
         border: false,
         viewConfig: {
-            emptyText: 'No courses for selected term'
+            emptyText: 'No courses for selected term',
+            getRowClass: function(record){
+                const now = new Date();
+                let isActive = true;
+
+                if (record.get('StartDate')) {
+                    let startTime = new Date(Date.parse(record.get('StartDate')));
+                    console.log(startTime, 'start time', now);
+                    isActive = startTime && startTime.getTime() < now.getTime();
+                }
+                if (isActive && record.get('EndDate')) {
+                    let endTime = new Date(Date.parse(record.get('EndDate')));
+                    isActive = endTime && endTime.getTime() > now.getTime();
+                }
+
+                return isActive === false ? 'inactive-participant' : 'active-participant';
+            }
+        },
+        plugins: {
+            ptype: 'cellediting',
+            clicksToEdit: 1
         },
         store: {
-            model: 'SlateAdmin.model.course.Section',
+            model: 'SlateAdmin.model.course.SectionParticipant',
```

Comment:
Check out the `Slate.model.course.SectionParticipant` class, I've been gradually migrating SlateAdmin towards using upgraded models under the `slate-core-data` package. Each model has an aliased proxy that goes along with it, so you can easily redefine a store-specific proxy anywhere you use the model in order to give it its own param state or override some defaults. Since the model requires its proxy already, you can require the model and then just use the proxy name in overridden config

E.g., the model has:

```
    proxy: 'slate-courses-participants'
```

So you can do:

```
    proxy: {
        type: 'slate-courses-participants',
        include: ['Section.Schedule', 'Section.Location']
    }
```

(apikit's records proxies have special support for dedicated `includes` and `relatedTables` configs)

If you can eliminate the last uses of SlateAdmin's local copy of that model, delete it